cpp/util/masterelection.cc

```diff
@@ -43,6 +43,28 @@ string EnsureEndsWithSlash(const string& s) {
 }  // namespace
 
 
+std::ostream& operator<<(std::ostream& output,
+                         MasterElection::ProposalState state) {
+  switch (state) {
+#define MY_PRINT_OUT_PROPOSALSTATE(state)    \
```

Comment:
Want me to repeat it "out long"? I guess putting it on one line should make it "not too bad", but it's repetitive (even within a line)...